code/game/jobs/job/civilians/other/survivors.dm

```diff
@@ -59,23 +61,31 @@
 
 	if(picked_spawner.story_text)
 		story_text = picked_spawner.story_text
+
+	if(picked_spawner.hostile)
+		hostile = TRUE
+
 	new /datum/cm_objective/move_mob/almayer/survivor(H)
 
-/datum/job/civilian/survivor/generate_entry_message(mob/living/carbon/human/H)
+/datum/job/civilian/survivor/generate_entry_message(mob/living/carbon/human/survivor)
 	if(intro_text)
+		if(hostile)
+			to_chat(survivor, SPAN_HIGHDANGER("You are HOSTILE to the USCM!"))
+		else
+			to_chat(survivor, SPAN_XENOHIGHDANGER("You are NON-HOSTILE to the USCM!"))
 		for(var/line in intro_text)
```

Comment:
With how much junk the survs get told here, put the you are hostile/nonhostile below this